packages/cpp/src/browser/cpp-preferences.ts

```diff
@@ -50,11 +50,13 @@ export const cppPreferencesSchema: PreferenceSchema = {
                     directory: ''
                 }
             ],
+            scopes: PreferenceScope.Default & PreferenceScope.User & PreferenceScope.Workspace
         },
         'cpp.experimentalCommands': {
             description: 'Enable experimental commands mostly intended for Clangd developers.',
             default: false,
-            type: 'boolean'
+            type: 'boolean',
+            scopes: PreferenceScope.Default & PreferenceScope.User & PreferenceScope.Workspace
```

Comment:
Could you make `scopes` optional, use `PreferenceScope.Default & PreferenceScope.User & PreferenceScope.Workspace` as default, revert changes to all preferences, only apply custom `scopes` where it is necessary? Otherwise it is a breaking change.

Reply:
you are right ! in the latest version of code i made it optional.

Reply:
What is the default?